app/utils/computed-helpers.js

```diff
@@ -14,19 +14,33 @@ export const computedSegmentedLink = function(property) {
   return computed(property, {
     get() {
       const splitted = this.get(property) ? this.get(property).split('://') : [];
+      const socialPlatforms = ['twitter', 'facebook', 'instagram', 'linkedin', 'youtube', 'github', 'gitlab', 'vimeo', 'flicker'];
```

Comment:
Move this up and export this and import it in the component above to remove duplication

Reply:
Done